diff --git a/notification_center.py b/notification_center.py
index 257ba0a..7437799 100644
--- a/notification_center.py
+++ b/notification_center.py
@@ -15,11 +15,7 @@ SCRIPT_DESC = 'Pass highlights and private messages to the macOS Notification Ce
 
 weechat.register(SCRIPT_NAME, SCRIPT_AUTHOR, SCRIPT_VERSION, SCRIPT_LICENSE, SCRIPT_DESC, '', '')
 
-WEECHAT_VERSION = weechat.info_get('version_number', '') or 0
-if int(WEECHAT_VERSION) >= 0x03020000:
-	WEECHAT_ICON = os.path.join(weechat.info_get('weechat_config_dir', ''), 'weechat.png')
-else:
-	WEECHAT_ICON = os.path.join(weechat.info_get('weechat_dir', ''), 'weechat.png')
+WEECHAT_ICON = '@out@/share/weechat.png'
 
 DEFAULT_OPTIONS = {
 	'show_highlights': 'on',
